topk-js/tests/expressions.spec.ts

```diff
@@ -0,0 +1,103 @@
+import { field, literal } from '../query';
+
+describe('Expression Tests', () => {
+  describe('Query expressions with flexible expressions', () => {
+    test('addition operations', () => {
+      expect(field('a').add(1).expr).toEqual(field('a').add(literal(1)).expr);
+      // JS doesn't support operator overloading, so we can't test 1 + field('a')
```

Comment:
makes sense, can you only type this once at the top instead of repeating in every fn?